.github/workflows/packages.yml

```diff
@@ -6,6 +6,7 @@ on:
     - master
     - dev
     - 'dev/**'
+    - 'gnucobol-db'
```

Comment:
Do not change the CI workflow file. You can also create a Pull Request in your own fork for CI testing.